test/functional/FunctionalTests.cpp

```diff
@@ -148,13 +149,121 @@ TEST_CASE_METHOD(
 
     // Check to see if we've received the relay message
     std::unique_lock<std::mutex> lock(recvRelayMutex);
-    recvRelayCv.wait_for(lock, std::chrono::milliseconds(5000));
+    recvRelayCv.wait_for(lock, std::chrono::milliseconds(WAIT_TIMEOUT_MS));
```

Comment:
nit: I prefer to strongly type the constant itself when feasible rather than depend on a naming convention.

Reply:
Done!